### DISCUSSION TITLE
Advanced Object Search doesn't find all objects

### DISCUSSION THREAD

**Discussion by TS1997** [pimcore/pimcore#12495] [Q&A / Support]
Hello,

I installed Pimcore Advanced Object Search in order to implement search functionality on a site but the results I get are not what I expect. A subset of the data object with the fields that should be searchable looks as follows:
![image](https://user-images.githubusercontent.com/5517212/175251801-0c2b2703-5885-43bb-a400-b91df5e81336.png)

If I search for the word "banderoll" using the built-in Pimcore admin search I get the following result
![image](https://user-images.githubusercontent.com/5517212/175252090-a1aa2065-b848-4517-a3ac-abf4486d34f4.png)
but when I do a search in Advanced Object Search I don't get nearly the same amount of results. When I only enter the word "banderoll" as a search term I get the following
![image](https://user-images.githubusercontent.com/5517212/175252553-0f652014-aee6-4b36-ac6f-18d688888df5.png)
and if I do it using the filters in the picture below with or without the search term I get 0 results. 
![image](https://user-images.githubusercontent.com/5517212/175253059-7f21f7a2-3847-4d37-ac4c-26646f615bab.png)

What am I doing wrong and how do I construct a search query that actually get the expected results? I've verified that the objects have been indexed correctly but am unable to search them

The goal is to be able to find all objects containing the search term, preferably sorted by relevance but I'm unfortunately unable to do so

**Comment by kingjia90** [comment]
I am a noob in ElasticSearch, but my 2 cents, shouldn't be all these conditions under a `Group` filter with Operator `should`?

**Reply by TS1997** [reply]
Thanks for the suggestion but that also resulted in 0 results unfortunately using the following config
![image](https://user-images.githubusercontent.com/5517212/175290694-7a20b6b3-e633-4387-8d80-5c22c7285e1f.png)
